Test case test_pvc_expand_expanded_pvc failed due to timeout while installing fio

Timeout occurred while waiting for fio setup to be completed. It was observed manually that sometimes the set up will take more than 180 seconds to complete. So the wait time should be updated to a higher value.

```
 Stack Trace

self = <tests.manage.pv_services.pvc_resize.test_pvc_expansion.TestPvcExpand object at 0x7f3d48e0bda0>

    @pytest.mark.polarion_id('OCS-302')
    def test_pvc_expand_expanded_pvc(self):
        """
        Verify PVC expand of already expanded PVC
    
        """
        pvc_size_expanded_1 = 20
        pvc_size_expanded_2 = 25
        executor = ThreadPoolExecutor(max_workers=len(self.pods))
    
        # Do setup on pods for running IO
        log.info("Setting up pods for running IO.")
        for pod_obj in self.pods:
            log.info(f"Setting up pod {pod_obj.name} for running IO")
            if pod_obj.pvc.volume_mode == 'Block':
                storage_type = 'block'
            else:
                storage_type = 'fs'
            executor.submit(pod_obj.workload_setup, storage_type=storage_type)
    
        # Wait for setup on pods to complete
        for pod_obj in self.pods:
            log.info(
                f"Waiting for IO setup to complete on pod {pod_obj.name}"
            )
            for sample in TimeoutSampler(
>               180, 2, getattr, pod_obj, 'wl_setup_done'
            ):

tests/manage/pv_services/pvc_resize/test_pvc_expansion.py:198: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <ocs_ci.utility.utils.TimeoutSampler object at 0x7f3d47169518>

    def __iter__(self):
        if self.start_time is None:
            self.start_time = time.time()
        while True:
            self.last_sample_time = time.time()
            try:
                yield self.func(*self.func_args, **self.func_kwargs)
            except Exception as ex:
                msg = f"Exception raised during iteration: {ex}"
                logging.error(msg)
            if self.timeout < (time.time() - self.start_time):
>               raise self.timeout_exc_cls(*self.timeout_exc_args)
E               ocs_ci.ocs.exceptions.TimeoutExpiredError: Timed Out: (180,)

ocs_ci/utility/utils.py:772: TimeoutExpiredError
```

--- a/tests/manage/pv_services/pvc_resize/test_pvc_expansion.py
+++ b/tests/manage/pv_services/pvc_resize/test_pvc_expansion.py
@@ -197,7 +197,7 @@ class TestPvcExpand(ManageTest):
                 f"Waiting for IO setup to complete on pod {pod_obj.name}"
             )
             for sample in TimeoutSampler(
-                180, 2, getattr, pod_obj, 'wl_setup_done'
+                360, 2, getattr, pod_obj, 'wl_setup_done'
             ):
                 if sample:
                     log.info(

